data_fetcher.py

```diff
@@ -1,18 +1,20 @@
 import datetime
 import time
-import itertools
 import json
 from db_connector import MongoAPI
 from utils import adex_calls
-
-
-adex_tickers = ["AWC", "AXE", "BAT", "BCH", "BET", "BOTS", "BTC", "BUSD", "CCL", "CHIPS", "CRYPTO", "DAI", "DASH",
-                "DEX", "DGB", "DOGE", "ECA", "EMC2", "ETH", "FTC", "HUSH", "ILN", "JUMBLR", "KMD", "LABS", "LTC",
-                "MCL", "MGW", "MORTY", "NAV", "OOT", "PANGEA", "PAX", "QTUM", "REVS", "RFOX", "RICK", "RVN",
-                "SUPERNET", "TUSD", "USDC", "VRSC", "XZC", "ZEC", "ZER"]
-
-# 45 tickers atm = 1980 pairs
-possible_pairs = list(itertools.permutations(adex_tickers, 2))
+from utils.util_funct import find_unique_pairs as get_pairs
+from utils.util_funct import enforce_float_type as enforce
+
+
+# adex_tickers = ["AWC", "AXE", "BAT", "BCH", "BET", "BOTS", "BTC", "BUSD", "CCL", "CHIPS", "CRYPTO", "DAI", "DASH",
+#                 "DEX", "DGB", "DOGE", "ECA", "EMC2", "ETH", "FTC", "HUSH", "ILN", "JUMBLR", "KMD", "LABS", "LTC",
+#                 "MCL", "MGW", "MORTY", "NAV", "OOT", "PANGEA", "PAX", "QTUM", "REVS", "RFOX", "RICK", "RVN",
+#                 "SUPERNET", "TUSD", "USDC", "VRSC", "XZC", "ZEC", "ZER"]
+#
+# 45 tickers atm = 1980 pairs // 283 in db as of 09.27
+# possible_pairs = list(itertools.permutations(adex_tickers, 2))
+possible_pairs = get_pairs()  # ~25s execution vs ~207s for above
```

Comment:
where can I find get_pairs function?

Reply:
utils/util_funct.py

Reply:
^^ 
`from utils.util_funct import find_unique_pairs as get_pairs`

Reply:
oh lol, not noticed imports